### DISCUSSION TITLE
Lock file problem

### DISCUSSION THREAD

**Discussion by TG-009** [community/community#42409] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am using git hub desktop and I am getting this message no matter what I do "A Lock file already exists in the repository, which blocks this operation from completing."
What do I do, I get this message every time I try to change branches, pull the origin.
HELP

**Comment by getlost01** [comment]
To resolve this issue, you'll need to delete the lock file and try the operation again. Here's how you can do that:

1. Close the GitHub Desktop app and any other processes that might be accessing the repository.
2. Open a terminal window and navigate to the root directory of your repository.
3. Run the following command to delete the lock file:
```
 git rm --cached -f .git/index.lock
 ```
4. Open the GitHub Desktop app and try the operation again.

If the issue persists, it's possible that there is another process that is accessing the repository and creating the lock file. In this case, you'll need to identify and close that process in order to delete the lock file.

**Reply by TG-009** [reply]
Thank you

**Reply by thisisabhinay** [reply]
On running the above command I am getting:
```
fatal: pathspec '.git/index.lock' did not match any files
```
So it's safe to conclude there's no `index.lockfile` exist in my case but why then I am still seeing this error? And how to fix it now?

**Reply by camjgodfrey** [reply]
> On running the above command I am getting:
> 
> ```
> fatal: pathspec '.git/index.lock' did not match any files
> ```
> 
> So it's safe to conclude there's no `index.lockfile` exist in my case but why then I am still seeing this error? And how to fix it now?

i couldn't find it with the terminal window command, but this worked for me...

go into Windows Explorer > navigate to the parent directory of your project > add \.git to the path > delete index.lock
